trestle/utils/load_distributed.py

```diff
@@ -24,40 +24,40 @@
 from trestle.utils import fs
 
 
-def _load_list(file_path: Path) -> Tuple[Type[OscalBaseModel], str, List[OscalBaseModel]]:
+def _load_list(abs_path: Path, abs_trestle_root: Path) -> Tuple[Type[OscalBaseModel], str, List[OscalBaseModel]]:
     """Given path to a directory of list(array) models, load the distributed models."""
     aliases_not_to_be_stripped = []
     instances_to_be_merged: List[OscalBaseModel] = []
-    file_path = file_path.resolve()
-    collection_model_type, collection_model_alias = fs.get_stripped_contextual_model(file_path)
-
-    for path in sorted(Path.iterdir(file_path)):
+    abs_path = abs_path.resolve()
```

Comment:
Kinda weird to call resolve on what is labeled an abs_path.  In that sense file_path is better.